src/devices/node/node.cpp

```diff
@@ -126,8 +126,8 @@ void setup() {
   // Documentation for this field here:
   // http://asf.atmel.com/docs/latest/samd20/html/group__asfdoc__sam0__wdt__group.html
   WDT->CONFIG.bit.PER =
-      9;  // WDT clock runs at ~32kHz (effective). A value of 9 means 4096
-          // cycles, which means a period of ~128ms.
+      0xB;  // WDT clock runs at ~32kHz (effective). A value of 0xB means 16384
+          // cycles, which means a period of ~512ms.
```

Comment:
@ademuri Is this okay? The watchdog was restarting the processor given it took so long to calculate a 256 length strip of LEDs.

Reply:
If that's the case, something else is going wrong - it should not take 128ms to run the main loop. So, I think you should debug why it's taking so long. It should take single-digit milliseconds to calculate the LEDs.